deep-learning-tools/templates/configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+type: Opaque
+data:
+  notebook_config.py: {{ include "notebook-config" . }}
```

Comment:
When processing 'configmap.yaml', the contents of 'notebook-config' is directly inserted (i.e .unquoted), so Helm attempts to parse the data as YAML, thus causing an error.

This can be fixed by replacing the line with
`notebook_config.py:  {{ include "notebook-config" . | quote }` :)

Reply:
Nice work! It otherwise looks (and works) nicely :)

You can bump the version in the Chart.yaml file as well, if you want

Reply:
I have fixed the quote issue and bumped the version 0.2.1